dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/enums/eutaxonomy/nonfinancials/Activity.kt

```diff
@@ -3,117 +3,161 @@ package org.dataland.datalandbackend.model.enums.eutaxonomy.nonfinancials
 import io.swagger.v3.oas.annotations.media.Schema
 
 /**
- * An enum that holds the different activites related to EU taxonomy for non financial companies
-*/
+ * An enum that holds the different activities related to EU taxonomy for non financial companies
+ */
 @Schema(
     enumAsRef = true,
 )
 enum class Activity(val value: String) {
+    AcquisitionAndOwnershipOfBuildings("Acquisition and ownership of buildings"),
```

Comment:
Are there any tests ensuring the consistency between Activity.kt, ActivityTree.ts and EuTaxonomyNames.ts?

Reply:
Also: Have any activities been removed or renamed with this change?

Reply:
There are no tests at the moment, the whole taxonomy stuff is rather hacky in DevTools Repo.

No activities have been removed or renamed.